mysql-test/include/ctype_utf8mb4.inc

```diff
@@ -1709,7 +1709,7 @@ SELECT count(*) FROM t1, t2
 # Alter from 4-byte charset to 3-byte charset, error
 --disable_warnings
 SET STATEMENT sql_mode = 'NO_ENGINE_SUBSTITUTION' FOR
-ALTER TABLE t1 CONVERT TO CHARACTER SET utf8;
+ALTER TABLE t1 CONVERT TO CHARACTER SET utf8, ALGORITHM=COPY;
```

Comment:
Instead of changing the existing test, try to extend it.
For example,
```diff
+set alter_algorithm=copy;
 SET STATEMENT sql_mode = 'NO_ENGINE_SUBSTITUTION' FOR
 ALTER TABLE t1 CONVERT TO CHARACTER SET utf8;
+set alter_algorithm=default;
+SET STATEMENT sql_mode = 'NO_ENGINE_SUBSTITUTION' FOR
+ALTER TABLE t1 CONVERT TO CHARACTER SET utf8;
```
The operation with `ALGORITHM=INPLACE` should report warnings or errors if and only if `ALGORITHM=COPY` does. Failure to keep full compatibility could break replication and cause corruption.